From d228a89afe97da43e3f03cd9c29509497b06336f Mon Sep 17 00:00:00 2001
From: Ben Walton <bwalton@opencsw.org>
Date: Fri, 27 Nov 2009 03:22:18 +0100
Subject: [PATCH] Update Paths and Binary Tool Names for CSW

/usr/bin/perl -> /opt/csw/bin/perl
export PATH=/opt/csw/bin:$PATH
make -> gmake
stty -> gstty

misc scripting cleanups.

Signed-off-by: Ben Walton <bwalton@opencsw.org>
---
 clmake       |    6 +++++-
 cmake        |    7 ++++++-
 colormake.pl |    2 +-
 3 files changed, 12 insertions(+), 3 deletions(-)

diff --git a/clmake b/clmake
index c091e23..18c8758 100755
--- a/clmake
+++ b/clmake
@@ -4,4 +4,8 @@
 # Jumps to the first gcc error that occurs during the build process.
 #
 
-make $* 2>&1 | colormake.pl `stty size` | less -r -pError
+export PATH=/opt/csw/bin:$PATH
+
+size=$(gstty size)
+
+gmake $* 2>&1 | colormake.pl $size | less -r -pError
diff --git a/cmake b/cmake
index 303e91e..6d3a674 100755
--- a/cmake
+++ b/cmake
@@ -4,4 +4,9 @@
 # Jumps to the first gcc error that occurs during the build process.
 #
 
-make $* 2>&1 | colormake.pl `stty size`
+export PATH=/opt/csw/bin:$PATH
+
+size=$(gstty size)
+
+gmake $* 2>&1 | colormake.pl $size
+
diff --git a/colormake.pl b/colormake.pl
index 1d9edb2..7d7b7d2 100755
--- a/colormake.pl
+++ b/colormake.pl
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/opt/csw/bin/perl
 #
 # colormake.pl 0.3
 #
-- 
1.6.5.1

